src/middlewares/blockUser.js

```diff
@@ -0,0 +1,25 @@
+import Model from '../models';
+
+const { User } = Model;
+
+/**
+ * @param {object} req
+ * @param {object} res
+ * @returns {object} an object containing the information of all the users or null
+ */
+
+const blockUser = (req, res)=>{
+    User.update(
+        {isBlocked: true},
+        {where: {_id: req.params.id}}
```

Comment:
`{where: {userId: req.params.id}}`

There is nothing like `_id` in the `user model`, if you already put `_id` there, kindly remove it. as we use UUID for our db data so anyone don't guess a userId